### PR TITLE
Arms2

### PR DESCRIPTION
**Pull request by ThirstyCorgi**
We created an arm subsystem with a constructor that initializes the arm motor using motion magic voltage, the arm bore encoder, the pizza box motor, and the servo motor.  We also created a method to help set up our arm angle called "setAngle" that helps swing our arm. We have a method to spin the pizza box motor. We have commands to swing the arm forward and back, spin and stop the pizza box motor, and spin the servo motor. We have tested all of our commands on a roadkill and they seemed to work properly. The button binds have not yet been determined.

### CODE REVIEW

**Review by Chvoong24 — CHANGES_REQUESTED** [review]
Fix shuffleboard names, looks good.

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -45,7 +48,10 @@ public class Robot extends TimedRobot {
   public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and use the subsystems needed
     // for the specific robot
-
+    //Draft
+    var talonFXConfigs = new TalonFXConfiguration();
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/Robot.java]: This is probably the wrong place for this.   Should it be in some subsystem?

File: src/main/java/frc/robot/commands/Arm/StopNoteContainerMotor.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final ArmSubsystem armSubsystem;
+
+  public StopNoteContainerMotor(ArmSubsystem armSubsystem) {
+    this.armSubsystem = armSubsystem;
+    addRequirements(armSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/Arm/StopNoteContainerMotor.java:22]: You can omit empty methods like this - there is an empty default in the super-class

File: src/main/java/frc/robot/commands/Arm/StopNoteContainerMotor.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    armSubsystem.stopPizzaBoxMotor();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/Arm/StopNoteContainerMotor.java:33]: Same here

**Review by RobertTheBlair — APPROVED** [review]
Some improvements, but can be fixed in the future

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (ThirstyCorgi) [reply src/main/java/frc/robot/Robot.java]: I agree. Will fix.

**Review by Chvoong24 — APPROVED** [review]

**Pull request merged**